index.js

```diff
@@ -143,8 +143,12 @@ function Construct(options, callback) {
           }
           fs.writeSync(out, page.slug);
         } else {
+          var pageLevel = (1.0 - page.level / 10);
```

Comment:
This would be clearer...

`var pageLevel = 1.0 - (page.level / 10)`

Reply:
It should run just fine with standard JS. I don't disagree it'd be clearer as you've written, though. To confirm for myself I checked the MDN and OOO is as expected: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Operator_Precedence#Table.

Reply:
Yes, it's correct, but our house style is to use parens when there's even a whisper of doubt (:

Reply:
(I have been doing this for mumbletypeg years and I still load up on parens by choice)